eva_submission/etc/eva_project_validation.yaml

```diff
@@ -7,11 +7,11 @@ Submitter Details:
     schema:
       Last Name: {         type: 'string',  nullable: False, required: True}
       First Name: {        type: 'string',  nullable: False, required: True}
-      Telephone Number: {  nullable: False, required: True}
+      Telephone Number: {  nullable: True, required: False}
       Email Address: {     type: 'string',  nullable: False, required: True}
       Laboratory: {        type: 'string',  nullable: False, required: True}
       Center: {            type: 'string',  nullable: False, required: True}
-      Address: {           type: 'string',  nullable: False, required: True}
+      Address: {           type: 'string',  nullable: True, required: False}
```

Comment:
Did you confirm the brokering doesn't need these?

Reply:
Yes. It turns out that the Phone number is not used and Address is only used if present: See [here](https://github.com/EBIvariation/eva-submission/blob/d5261302fedff68259b0b9940a45c20babf690c3/eva_submission/biosample_submission/biosamples_submitters.py#L347)